roles/zuul/files/jobs/lore.yaml

```diff
@@ -0,0 +1,14 @@
+- job:
+    name: lore
+    node: 'ubuntu-xenial'
+
+    builders:
+      - zuul-git-prep
+      - shell: |
+        #!/bin/bash -ex
```

Comment:
@eggshell pointed out that these should be indented by four spaces (rather than two as they are here) to match the other files in roles/zuul/files/jobs/.